lib/kubeclient/common.rb

```diff
@@ -405,6 +405,17 @@ def proxy_url(kind, name, port, namespace = '')
       end
     end
 
+    def process_template(template)
+      ns_prefix = build_namespace_prefix(template[:metadata][:namespace])
+      @headers['Content-Type'] = 'application/json'
```

Comment:
Actually we do not want to update @headers except for this request.
Please follow https://github.com/abonas/kubeclient/pull/190 to clean this up.